src/components/gltf-model.js

```diff
@@ -22,7 +22,7 @@ module.exports.Component = registerComponent('gltf-model', {
     this.remove();
 
     this.loader.load(src, function gltfLoaded (gltfModel) {
-      self.model = gltfModel.scene;
+      self.model = gltfModel.scene || gltfModel.scenes[0];
```

Comment:
Yes, in practice I haven't found multiple scenes in a glTF yet, but there's no requirement that `scene` === `scenes[0]`, or that `scene` even be provided.

The change looks good; we could also add a `scene:0` property to the `gltf-model` schema to select one, if that becomes a thing.